scripts/vendor-clone.sh

```diff
@@ -5,5 +5,5 @@ if [ -d "src/vendor/react-store" ]; then
     echo "Skipping react-store (already cloned)"
 else
     echo "Cloning react-store to src/vendor/react-store"
-    git clone git@github.com:toggle-corp/react-store.git src/vendor/react-store
+    git clone https://github.com/toggle-corp/react-store.git src/vendor/react-store
```

Comment:
@ShrawanBK , was this required?
If not, let's revert it back.

Reply:
It's not really required!